en/CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing guidelines for Chinese-to-English
+
+We are working on translating "Hello Algo" from Chinese to English with the following approach:
+
+1. **AI translation**: Carry out an initial pass of translations using the machine learning translator.
+2. **Human optimization**: Manually refine the machine-generated outputs to ensure authenticity and accuracy.
+3. **Pull request review**: The optimized translation will be doubly checked by the reviewers through GitHub pull request workflow.
+4. Repeat steps `2.` and `3.` for further improvements.
+
+## Join us
+
+We're seeking contributors who meet the following criteria.
+
+- **Technical background**: Strong foundation in computer science, particularly in data structures and algorithms.
+- **Language skills**: Native proficiency in Chinese with professional-level English, or native English.
+- **Available time**: Dedicated to contributing to open-source projects with a willingness to engage in long-term translation efforts.
+
+That is, our contributors are computer scientists, engineers, and students from different linguistic backgrounds, and their objectives have different focal points:
+
+- **Native Chinese with professional working English**: Ensuring translation accuracy and consistency between CN and EN versions.
+- **Native English**: Enhance the authenticity and fluency of the English content to read natural and engaging.
+
+Don't hesitate to join us via WeChat `krahets-jyd` or on [Discord](https://discord.gg/9hrbyZFBX3)!
+
+## Translation process
+
+> [!important]
+> Before diving in, ensure you're comfortable with the GitHub pull request workflow and have read the "Translation standards" and "Pseudo-code for translation" below.
+
+1. **Self-assignment**: Visit [GitHub projects](https://github.com/users/krahets/projects/2/views/4) to select an unclaimed task and mark it as `In Progress`.
+2. **Translation**: We encourage preserving the original meaning while ensuring the translation is natural and fluent.
+3. **Peer review**: Please carefully check your changes before submitting a Pull Request (PR). After approval by two reviewers, it will be merged into the project.
+
+## Translation standards
+
+> [!tip]
+> The "Accuracy" and "Authenticity" are primarily handled by native Chinese speakers and native English speakers, respectively.
+
+**Accuracy**:
+
+- Maintain consistency in terminology across translations by referring to the [Terminology](https://www.hello-algo.com/chapter_appendix/terminology/) section.
+- Prioritize technical accuracy and maintain the tone and style of the Chinese version. If you think changing the original meaning is necessary, please first submit a PR to modify the Chinese version.
+
+**Authenticity**:
+
+- Translations should read natural and fluent, adhering to English expression conventions.
```

Comment:
'natural' and 'fluent' should be adverbs (they describe the verb 'read'). I suggest "Translations should read naturally and fluently, adhering to English expression conventions."